tensorflow/inference/docker/2.9/py3/Dockerfile.ec2.graviton.cpu.os_scan_allowlist.json

```diff
@@ -0,0 +1,974 @@
+      "source_url": "https://people.canonical.com/~ubuntu-security/cve/2022/CVE-2022-32547.html",
+      "source": "UBUNTU_CVE",
+      "severity": "MEDIUM",
+      "status": "ACTIVE",
+      "title": "CVE-2022-32547 - imagemagick-6-common, libmagickcore-6.q16-6 and 3 more"
+    },
+    {
+      "description": " In ImageMagick, there is load of misaligned address for type 'double', which requires 8 byte alignment and for type 'float', which requires 4 byte alignment at MagickCore/property.c. Whenever crafted or untrusted input is processed by ImageMagick, this causes a negative impact to application availability or other problems related to undefined behavior.",
+      "vulnerability_id": "CVE-2022-32547",
+      "name": "CVE-2022-32547",
+      "package_name": "imagemagick",
+      "package_details": {
+        "file_path": null,
+        "name": "imagemagick",
+        "package_manager": "OS",
+        "version": "6.9.10.23+dfsg",
+        "release": "2.1ubuntu11.4"
+      },
+      "remediation": { "recommendation": { "text": "None Provided" } },
+      "cvss_v3_score": 7.8,
+      "cvss_v30_score": 0.0,
+      "cvss_v31_score": 7.8,
+      "cvss_v2_score": 6.8,
+      "cvss_v3_severity": "HIGH",
+      "source_url": "https://people.canonical.com/~ubuntu-security/cve/2022/CVE-2022-32547.html",
+      "source": "UBUNTU_CVE",
+      "severity": "MEDIUM",
+      "status": "ACTIVE",
+      "title": "CVE-2022-32547 - imagemagick-6-common, libmagickcore-6.q16-6 and 3 more"
+    },
+    {
+      "description": " A vulnerability was found in ImageMagick, causing an outside the range of representable values of type 'unsigned char' at coders/psd.c, when crafted or untrusted input is processed. This leads to a negative impact to application availability or other problems related to undefined behavior.",
+      "vulnerability_id": "CVE-2022-32545",
+      "name": "CVE-2022-32545",
+      "package_name": "imagemagick",
+      "package_details": {
+        "file_path": null,
+        "name": "imagemagick",
+        "package_manager": "OS",
+        "version": "6.9.10.23+dfsg",
+        "release": "2.1ubuntu11.4"
+      },
+      "remediation": { "recommendation": { "text": "None Provided" } },
+      "cvss_v3_score": 7.8,
+      "cvss_v30_score": 0.0,
+      "cvss_v31_score": 7.8,
+      "cvss_v2_score": 6.8,
+      "cvss_v3_severity": "HIGH",
+      "source_url": "https://people.canonical.com/~ubuntu-security/cve/2022/CVE-2022-32545.html",
+      "source": "UBUNTU_CVE",
+      "severity": "MEDIUM",
+      "status": "ACTIVE",
+      "title": "CVE-2022-32545 - imagemagick-6-common, libmagickcore-6.q16-6 and 3 more"
+    },
+    {
+      "description": " A vulnerability was found in ImageMagick, causing an outside the range of representable values of type 'unsigned char' at coders/psd.c, when crafted or untrusted input is processed. This leads to a negative impact to application availability or other problems related to undefined behavior.",
```

Comment:
Same record duplication and omission of reason_to_ignore observed. Please review allowlist.json.

Reply:
same response as above.